tools/dev/k3d/Makefile

```
@@ -66,11 +58,20 @@ secrets: secrets/grafana.jwt secrets/gel.jwt
 
 secrets/grafana.jwt:
 	mkdir -p secrets/
-	op document get "loki/grafana.jwt" --output=$(CURDIR)/secrets/grafana.jwt
+	op document get "loki/grafana.jwt" --output=$(CURDIR)/secrets/grafana.jwt || rm -f $(CURDIR)/secrets/grafana.jwt
+	@if ! test -s "$(CURDIR)/secrets/grafana.jwt"; then \
+		echo "Failed to fetch Grafana license from 1Password, are you logged in?"; \
+		rm -f $(CURDIR)/secrets/grafana.jwt; \
+		exit 1; \
+	fi
 
 secrets/gel.jwt:
 	mkdir -p secrets/
-	op document get "loki/gel.jwt" --output=$(CURDIR)/secrets/gel.jwt
+	op document get "loki/gel.jwt" --output=$(CURDIR)/secrets/gel.jwt || rm -f $(CURDIR)/secrets/gel.jwt
+	@if ! test -s "$(CURDIR)/secrets/gel.jwt"; then \
+		echo "Failed to fetch GEL license from 1Password, are you logged in?"; \
```

We should also rm the gel license file in this case, right? As it might be empty:
```suggestion
		echo "Failed to fetch GEL license from 1Password, are you logged in?"; \
		rm -f $(CURDIR)/secrets/gel.jwt; \
```

---

yes, thank you!